### PR TITLE
Replaced building locally section with documentation section

### PR DESCRIPTION
**Pull request by evaherrada**
@kattni

### CODE REVIEW

**Review by kattni — CHANGES_REQUESTED** [review]
@dherrada I'm not sure why you closed the other one to open this one. We still need the changes requested on the other one. It needs PyPI instructions, and it needs the Installation section removed. You can make the changes on this PR.

**Comment by evaherrada** [comment]
@kattni Sorry about that. I got confused and thought you were referring to removing the "installing from pypi" section instead of the "installation" section.

File: README.rst [hunk 1]
```diff
@@ -20,18 +20,26 @@ changes such as mode. For I2C, it manages the device address.
...
-To install:
-
-#. Download and unzip the `latest release zip <https://github.com/adafruit/Adafruit_CircuitPython_BusDevice/releases>`_.
-#. Copy the unzipped ``adafruit_bus_device`` to the ``lib`` directory on the ``CIRCUITPY`` drive.
+.. code-block:: shell
+    
+    pip3 install adafruit-circuitpython-busdevice
+    
+To install system-wide (this may be required in some cases):
+.. code-block:: shell
```
Comment (kattni) [inline README.rst:32]: There should be a blank line between this and the line above it.

File: README.rst [hunk 2]
```diff
@@ -20,18 +20,26 @@ changes such as mode. For I2C, it manages the device address.
...
+    
+    sudo pip3 install adafruit-circuitpython-busdevice
+   
+To install in a virtual environment in your current project:
+.. code-block:: shell
```
Comment (kattni) [inline README.rst:38]: There should be a blank line between this and the line above it.

**Review by kattni — CHANGES_REQUESTED** [review]
Thanks for adding this! There are two whitespace issues, see inline comments. Get those fixed up and it's ready to merge.

**Review by kattni — APPROVED** [review]

**Pull request merged**